orchestrator/forms/validators.py

```diff
@@ -293,29 +273,6 @@ def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
         field_schema.update(format="label", type="string")
 
 
-class MigrationSummary(DisplayOnlyFieldType):
```

Comment:
Isnt this generic? There is a frontend component for it as well

Reply:
Moved to `network_type_validators`